espresso-support/demo/src/main/java/com/novoda/movies/MovieItemView.java

```diff
@@ -1,52 +1,32 @@
 package com.novoda.movies;
 
 import android.content.Context;
-import android.support.v4.view.ViewCompat;
-import android.support.v7.app.AlertDialog;
 import android.util.AttributeSet;
 import android.view.View;
 import android.widget.RelativeLayout;
 import android.widget.TextView;
 
-import com.novoda.accessibility.AccessibilityServices;
-import com.novoda.accessibility.Action;
-import com.novoda.accessibility.Actions;
-import com.novoda.accessibility.ActionsAccessibilityDelegate;
-import com.novoda.accessibility.ActionsAlertDialogCreator;
-
-import java.util.Arrays;
-import java.util.Locale;
-
-import butterknife.BindView;
-import butterknife.ButterKnife;
-
 public class MovieItemView extends RelativeLayout {
```

Comment:
The changes in this class should be in the base branch, dunno why they're showing up. It's just removing ButterKnife and Accessibilitools to make the demo simpler.